### DISCUSSION TITLE
Docs site

### DISCUSSION THREAD

**Discussion by jasonappah** [hackclub/hack-as-a-service-v0#131] [General]
So... we need a docs site. How do y'all want to build this, and what content should it have? I think at the minimum it should have a Getting Started guide that walks people through prepping your app for deployment, and actually getting it running.

**Comment by sampoder** [comment]
I have Nextra set up on local dev rn

**Comment by sampoder** [comment]
I could push?

**Reply by jasonappah** [reply]
Oh that'd be awesome

**Comment by jasonappah** [comment]
Thanks @sampoder for getting this running!

**Comment by anirudhb** [comment]
Fully setup as of #138
